examples/web-app/src/components/App.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Navbar from "./Navbar/Navbar";
+import Home from "./Home/Home";
+import About from "./About/About";
+import "bootstrap/dist/css/bootstrap.css";
```

Comment:
Again, not a mistake, because this one can be injected pretty much anywhere, but personally i would stick this one on `index.js`.

Reply:
Yes i know that this also can be the convention but I decided to use App.js like sort of an index.js file.